### PR TITLE
Cleanup extended test output some more

### PR DESCRIPTION
**Pull request by mnagy**
Write to GinkgoWriter and use upstream e2e.Logf()

@stevekuznetsov

### CODE REVIEW

**Comment by mnagy** [comment]
[testonlyextended][extended:core]

**Comment by stevekuznetsov** [comment]
LGTM, much better to have everything in the e2e logger

File: test/extended/util/cli.go [hunk 1]
```diff
@@ -294,8 +295,8 @@ func (c *CLI) OutputToFile(filename string) (string, error) {
 // This function will set the default output to stdout.
```
Comment (stevekuznetsov) [inline test/extended/util/cli.go]: Outdated comment

**Comment by mfojtik** [comment]
+1 LGTM

**Comment by mnagy** [comment]
@stevekuznetsov good catch, fixed.

**Comment by mnagy** [comment]
Lots of failures.. These seem to be unrelated to the PR.

**Comment by mfojtik** [comment]
LGTM [merge]

**Comment by mnagy** [comment]
Fails because of #5937

**Comment by mfojtik** [comment]
re [merge]

**Pull request merged**